pkg/networkservice/chains/nsmgr/suite_test.go

```diff
@@ -405,7 +407,9 @@ func (s *nsmgrSuite) Test_PassThroughRemoteUsecase() {
 					connect.NewServer(ctx,
 						sandbox.NewCrossConnectClientFactory(sandbox.GenerateTestToken,
 							newPassTroughClient(fmt.Sprintf("my-service-remote-%v", i-1)),
-							kernel.NewClient()),
+							kernel.NewClient(),
```

Comment:
Please add here `checkrequest.NewClient()` to validate that connection context is already set on refresh request.